core/src/test/java/org/elasticsearch/indices/IndicesOptionsIntegrationIT.java

```diff
@@ -669,14 +671,13 @@ public void testPutAlias_wildcard() throws Exception {
         assertThat(client().admin().indices().prepareAliasesExist("foobar_alias").setIndices("foobar").get().exists(), equalTo(true));
         assertThat(client().admin().indices().prepareAliasesExist("foobar_alias").setIndices("bar").get().exists(), equalTo(true));
         assertThat(client().admin().indices().prepareAliasesExist("foobar_alias").setIndices("barbaz").get().exists(), equalTo(true));
-        
+
     }
-    
+
     @Test
     public void testDeleteWarmer() throws Exception {
-        IndexWarmersMetaData.Entry entry = new IndexWarmersMetaData.Entry(
-                "test1", new String[]{"typ1"}, false, new BytesArray("{\"query\" : { \"match_all\" : {}}}")
-        );
+        SearchSourceBuilder source = new SearchSourceBuilder();
```

Comment:
maybe set match_all query explicitly to the search source builder to make the test more readable?